### DISCUSSION TITLE
You can optimize Lookup performance in row.cs by using ServiceLookupEditor instead of LookupEditor

### DISCUSSION THREAD

**Discussion by ibrahimGit123** [serenity-is/Serenity#6680] [Tricks and Samples]
Just a tip-  You can optimize Lookups performance in row.cs to run much faster for large tables (thousands of rows) by using ServiceLookupEditor attribute instead of LookupEditor attribute.

Doing this helped reduce the load time of my grids and dialogs by a significant amount of time.

**Comment by gfo2007** [comment]
@ibrahimGit123 could you please provide example code of how to use ServiceLookupEditor?
